gollm/gemini.go

```diff
@@ -191,7 +192,13 @@ var _ Client = &GoogleAIClient{}
 
 // ListModels lists the models available in the Gemini API.
 func (c *GoogleAIClient) ListModels(ctx context.Context) (modelNames []string, err error) {
-	for model, err := range c.client.Models.All(ctx) {
+	next, stop := iter.Pull2(c.client.Models.All(ctx))
+	defer stop()
+	for {
+		model, err, ok := next()
+		if !ok {
+			break
+		}
```

Comment:
Can we pl. limit the changes to just whatever is needed for supporting the skip-ssl-verify ?

The main reason is that we don't have robust automated tests yet, so scoping the change to what is need will help in assessing the impact.

Reply:
@droot I will revert it.

Reply:
Done [force-pushed](https://github.com/GoogleCloudPlatform/kubectl-ai/compare/050f54eaa8efc40b5dd66785f7865fb66b883a5f..d9395b1f84e6b1f61991bfc15b0bda17ab6060f8) the skip branch from [050f54e](https://github.com/GoogleCloudPlatform/kubectl-ai/commit/050f54eaa8efc40b5dd66785f7865fb66b883a5f) to [d9395b1](https://github.com/GoogleCloudPlatform/kubectl-ai/commit/d9395b1f84e6b1f61991bfc15b0bda17ab6060f8)